README.md

```diff
@@ -1 +1,4 @@
 # sausage-project
+This is a website about sausages.
+
+Open index.html to view
```

Comment:
Change to `Open index.html to view` to `Open index.html to view in your browser`